src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const status = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age >= 0 && (
```

Comment:
Redundant, please simplify condition
```suggestion
      {age && (
```